Remove code deprecated in 2.5.0

njmulsqb opened the pull request:
Signed-off-by: Najam Ul Saqib <najamulsaqib@tutamail.com>

Related to #7069 

Help has been updated in https://github.com/zaproxy/zap-core-help/pull/420

psiinon commented on zap/zap.gradle.kts:
```diff
@@ -201,6 +201,7 @@ val japicmp by tasks.registering(JapicmpTask::class) {
         "org.parosproxy.paros.network.ConnectionParam#getProxyChainSkipName()",
         "org.parosproxy.paros.network.ConnectionParam#setProxyChainSkipName(java.lang.String)",
         "org.parosproxy.paros.view.AbstractFrame#loadIconImages()",
+        "org.parosproxy.paros.control.Control#createAndOpenUntitledDb()",
```
This should be in alphabetical order. Other than that this PR looks good so far :)

kingthorin reviewed (commented):
Looks good to me

njmulsqb commented on zap/zap.gradle.kts:
```diff
@@ -201,6 +201,7 @@ val japicmp by tasks.registering(JapicmpTask::class) {
         "org.parosproxy.paros.network.ConnectionParam#getProxyChainSkipName()",
         "org.parosproxy.paros.network.ConnectionParam#setProxyChainSkipName(java.lang.String)",
         "org.parosproxy.paros.view.AbstractFrame#loadIconImages()",
+        "org.parosproxy.paros.control.Control#createAndOpenUntitledDb()",
```
Doesn't pressing "resolve conversation" auto apply the suggestion? :P 
Do I need to make these changes locally and then push it?

psiinon replied:
Yeah, I just made a comment, I didnt actually make the changes :)

psiinon replied:
So yes, make the changes locally and push them

njmulsqb replied:
Oh Okay, I will do it

njmulsqb replied:
Done

kingthorin commented:
Why do you keep merging?

njmulsqb commented:
> Why do you keep merging?

I specifically didn't merge, I just took the pull from remote branch as otherwise it was not allowing me to push

kingthorin commented:
Okay, we'll get this sorted out. Can you provide like a bullet summary of your workflow?

njmulsqb commented:
> Okay, we'll get this sorted out. Can you provide like a bullet summary of your workflow?

Sure.
- I create a separate branch I am working on
- Create the changes on the branch
- Commit and sign off the changes
- Take latest pull to get the fresh code
- Push the code and make PR using Github UI

kingthorin commented:
Okay skip this step "Take latest pull to get the fresh code"

If you need to update for some reason then fetch and rebase.

thc202 commented on zap/src/main/java/org/parosproxy/paros/control/Control.java:
```diff
@@ -87,6 +87,7 @@
 // ZAP: 2021/09/13 Added setExitStatus.
 // ZAP: 2021/11/08 Validate if mandatory add-ons are present.
 // ZAP: 2022/02/09 No longer manage the proxy, deprecate related code.
+// ZAP: 2022/02/24 Remove code deprecated in 2.5.0
```
I'd suggest using this sentence in all files, for consistency.

thc202 commented:
The commits should be fixed up.

njmulsqb commented:
I think I messed up something, I took pull to get the commits I did here to make the `// ZAP` tag consistent and it brought other commits as well.
I rebased all of them and fixed into one.
Is it okay?

njmulsqb commented:
> Okay skip this step "Take latest pull to get the fresh code"
> 
> If you need to update for some reason then fetch and rebase.

Sorry bud, I repeated the mistake I believe. :/

psiinon commented:
Looks like you've pulled in an unrelated PR :/

njmulsqb commented:
> Looks like you've pulled in an unrelated PR :/

How can I reverse this?

thc202 commented:
Do:
 - `git fetch upstream`
 - `git rebase upstream/main`

njmulsqb commented:
> Do:
> 
>     * `git fetch upstream`
> 
>     * `git rebase upstream/main`

Done

thc202 commented:
You should have just commit then, you can (force) push to your remote.

njmulsqb commented:
> You should have just one commit then, you can (force) push to your remote.

Wow, that worked like magic. I need to study the git commands you shared

thc202 commented:
Will fix the conflicts caused by the merge of other PR.

thc202 commented:
Thank you!

psiinon reviewed (approved):
Nice :)
